__tests__/app.test.js

```diff
@@ -59,42 +59,78 @@ describe("GET /api/topics", () => {
-      expect(Array.isArray(body)).toBe(false)
-      expect(body).toHaveProperty("author")
-      expect(body).toHaveProperty("title")
-      expect(body).toHaveProperty("article_id")
-      expect(body).toHaveProperty('body')
-      expect(body).toHaveProperty('topic')
-      expect(body).toHaveProperty('created_at')
-      expect(body).toHaveProperty('votes')
-      expect(body).toHaveProperty('article_img_url')
-      expect(article_id).toEqual(1)
-    })
+      .get("/api/articles/1")
+      .expect(200)
+      .then(({ body }) => {
+        const { article_id } = body;
+        expect(typeof body).toBe('object')
+        expect(Array.isArray(body)).toBe(false)
+        expect(body).toHaveProperty("author")
+        expect(body).toHaveProperty("title")
+        expect(body).toHaveProperty("article_id")
+        expect(body).toHaveProperty('body')
+        expect(body).toHaveProperty('topic')
+        expect(body).toHaveProperty('created_at')
+        expect(body).toHaveProperty('votes')
+        expect(body).toHaveProperty('article_img_url')
+        expect(article_id).toEqual(1)
+      })
   });
 
   test("400: return 'invalid id' message when requesting an id that is not in the database", () => {
     return request(app)
-    .get("/api/articles/999")
-    .expect(400)
-    .then(({body}) =>{
-      const {msg} = body;
-      expect(msg).toBe('invalid id')
-    })
+      .get("/api/articles/999")
+      .expect(400)
+      .then(({ body }) => {
+        const { msg } = body;
+        expect(msg).toBe('invalid id')
+      })
   });
 
   test("404: return 'bad request' message when requesting with an invalid input type", () => {
     return request(app)
-    .get("/api/articles/asd")
-    .expect(404)
-    .then(({body}) =>{
-      const {msg} = body;
-      expect(msg).toBe('bad request')
-    })
+      .get("/api/articles/asd")
+      .expect(404)
+      .then(({ body }) => {
+        const { msg } = body;
+        expect(msg).toBe('bad request')
+      })
   });
 
+});
+
+describe("GET /api/articles", () => {
+  test("200: return an array of article objects", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body: { articles } }) => {
+        expect(Array.isArray(articles)).toBe(true)
+        expect(articles.length).toBeGreaterThan(0)
+        articles.forEach(function (article) {
+          expect(Array.isArray(article)).toEqual(false)
+          expect(typeof article).toEqual('object')
+          expect(article).toHaveProperty("author")
+          expect(article).toHaveProperty("title")
+          expect(article).toHaveProperty("article_id")
+          expect(article).toHaveProperty("topic")
+          expect(article).toHaveProperty("created_at")
+          expect(article).toHaveProperty("votes")
+          expect(article).toHaveProperty("article_img_url")
+          expect(article).toHaveProperty("comment_count")
+          expect(article).not.toHaveProperty("body")
```

Comment:
This is fine, however there are better jest matchers out there for this such as expect.ObjectContaining